x/slashing/keeper/infractions.go

```diff
@@ -18,9 +18,6 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr cryptotypes.Addre
 
 	// fetch the validator public key
 	consAddr := sdk.ConsAddress(addr)
-	if _, err := k.GetPubkey(ctx, addr); err != nil {
```

Comment:
exactly the same letter for letter as what @vuong177 wrote, and informal told us was not needed.  They could have cherry picked this and chose not to.